server.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
```

Comment:
Instead of this python script a better approach would be to use an [entry point](https://github.com/RuggedPOD/ruggedpod-cli/blob/91757be8b0cf65e4f6992f8f55ca3c591e8cf2b9/setup.cfg#L17) in the setup.cfg